master/buildbot/master.py

```diff
@@ -125,6 +125,10 @@ def __init__(self, basedir, configFileName="master.cfg", umask=None):
         self.name = self.name.decode('ascii', 'replace')
         self.masterid = None
 
+    @property
+    def master(self):
+        return self
```

Comment:
Indeed.. isn't that pretty?

This is a recursive attribute, related to the one in BuildbotService. This one is the recursion stop condition